contents/tutorials/product-data-in-new-tab.md

```diff
@@ -0,0 +1,79 @@
+
+![Insight short ID](../images/tutorials/product-data-in-new-tab/short-id.png)
+
+## Formatting the API request
+
+Once you have all three pieces of information, we are going to use them to format an API request for the data Momentum Dash uses to display it. The API format is:
+
+```bash
+https://<ph_instance_address>/api/projects/<project_id>/insights/?personal_api_key=<personal_key>&short_id=<insight_short_id>
+```
+
+So if our instance address was `app.posthog.com`, project ID was `12345`, our personal API key was `phx_abcde`, and our insight short ID was `aAbBcC`, our API request would look like:
+
+```bash
+https://app.posthog.com/api/projects/12345/insights/?personal_api_key=phx_abcde&short_id=aAbBcC
+```
+
+Copy your formatted code and open up a new tab (assuming you have Momentum Dash set up) to move on to the next step.
+
+## Adding our metric in Momentum Dash
+
+Once you’ve signed up for Momentum Dash Plus and added the extension, your new tab page should look like this:
+
+![Momentum new tab](../images/tutorials/product-data-in-new-tab/new-tab.png)
+
+Hover your mouse in the top right corner, and an “add” button appears. Click it and select “Metric.” Next, choose “From a URL” and paste the API request you created in the last section into the URL field. If you formatted the API request properly, a drop-down appears that prompts you to “select an option.” 
+
+![Momentum API request](../images/tutorials/product-data-in-new-tab/momentum-api.png)
+
+When you click the drop-down, you get lots of options, you have to find the one that matches your metric (number) in PostHog. For my number chart, it was `result[0].result[0].aggregated_value` but it might also be `result[0].result[0].count` or another value. 
+
+> **Note:** It is likely easier to modify the insight in PostHog to provide a single value through a number chart than to reformat your API request to get that value.
+
+Select the option with the right value from the drop-down, add a name, and then click “Add.” Once done, you have an updating metric on your new tab page.
+
+![Metric in new tab](../images/tutorials/product-data-in-new-tab/metric.png)
+
+Now, every time you open a new tab, you see product data showing you how you are doing. This value updates every time you open a new tab. You can add more or change them following the same process. These key product metrics provide important reminders every time you open a new tab.
+
+## Further reading
+
+- Deciding what metric to add to your new tab page? Churn rate is a popular one. Here’s [a tutorial on how you calculate (and lower) churn rate with PostHog](/tutorials/churn-rate).
+- Care more about time on site or average session duration? [This tutorial helps you calculate session-based metrics](/tutorials/session-metrics).
+- Would rather have product metrics in Slack? See how you can set up [Slack and PostHog in our docs](/docs/integrate/webhooks/slack).
```

Comment:
```suggestion
- Would rather have product metrics in Slack? See how you can set up [Slack and PostHog in our docs](/docs/integrate/webhooks/slack).
- Curious about how Momentum Dash handles analytics? Check out [how Momentum Dash uses PostHog](/customers/momentumdash).
```

Feel free to cut this if you don't think it fits, I'd just like to work it in if possible.